index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    
+    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"> 
```

Comment:
This will increase loading time and vue material already provide layout APIs. So don't use bootstrap.
https://vuematerial.io/ui-elements/layout

Reply:
> This will increase loading time and vue material already provide layout APIs. So don't use bootstrap.
> https://vuematerial.io/ui-elements/layout

I removed bootstrap and tried the vue material but could find the apt properties there, thus used different properties. Please do have a look now.